test/contracts/rangepool.ts

```diff
@@ -720,12 +795,118 @@ describe('RangePool Tests', function () {
       signer: hre.props.alice,
       lower: '500',
       upper: '1000',
-      tokenAmount: BigNumber.from('4152939701311089823384'),
-      liquidityAmount: BigNumber.from('4152939701311089823384'),
+      tokenAmount: aliceLiquidity,
+      liquidityAmount: aliceLiquidity,
+      fungible: true,
+      balance0Increase: BigNumber.from('92198793754243677820'),
+      balance1Increase: BigNumber.from('99375039384589972731'),
+      revertMessage: '',
+    })
+
+    if (balanceCheck) {
+      console.log('balance after token0:', (await hre.props.token0.balanceOf(hre.props.rangePool.address)).toString())
+      console.log('balance after token1:', (await hre.props.token1.balanceOf(hre.props.rangePool.address)).toString())
+    }
+  })
+
+  it('pool - Should mint position inside the other and not steal fee share :: KEBABSEC', async function () {
+    const pool: PoolState = await hre.props.rangePool.poolState()
```

Comment:
Test case added to remediate the following:
https://hackmd.io/@psdoc/r1VSEC2l3#Critical-Skipping-position-fee-growth-snapshot-can-be-abused-to-drain-the-entire-pool